### ISSUE TITLE
CasperJS not scrapping many sites

### ISSUE THREAD

**Issue by JeevPRobinson** [n1k0/casperjs#1394]
I tried scrapping and auto filing form in eventbrite site (https://www.eventbrite.com/). As the first step, I tried to reach the site using the following code:

```
var casper = require('casper').create();
casper.start('https://www.eventbrite.com/');
casper.then(function() {
    console.log('The current location is ' + this.getCurrentUrl());
});
casper.run();
```
The response I get is : 
```
The current location is about:blank
```
When I tried to use the same examples given in the documentation (http://docs.casperjs.org/en/latest/quickstart.html), it gave me proper results. The example had the google, casperjs and phantomjs urls. Also it succeeded for w3schools, youtube etc.
```  
eg:    casper.start('http://docs.casperjs.org/en/latest/');
         response:  The current location is http://docs.casperjs.org/en/latest/
```
Whereas it also failed for Facebook, Twitter, Instagram etc.

Can anyone help me out? Thanks.

**Comment by steverob** [comment]
This is very weird. I tried this - 

```js
var casper = require('casper').create();

casper.start('https://eventbrite.com', function() {
        this.echo(this.getHTML());
        });

casper.run();
```

Response:
`<html><head></head><body></body></html>`

**Comment by mpranjic** [comment]
Eventbrite example works for me. Casper - beta3, Phantom 1.9.8

**Comment by steverob** [comment]
Here's my setup @mpranjic 

```
$ casperjs --version
1.1.0-beta3

$ phantomjs --version
1.9.0
```

Btw, it looks like an SSL issue with PhantomJS. Found from other issues that adding `--ssl-protocol=any` fixes it. They say its been fixed in PhantomJS 2.0. Not sure how its working for you :)

**Comment by mpranjic** [comment]
@steverob yeah, I am using ``` --ssl-protocol ``` flag :)
